entries/m/masterclass.com.json

```
@@ -0,0 +1,12 @@
+{
+    "Masterclass": {
+      "domain": "masterclass.com",
+      "contact": {
+        "twitter": "MasterClass"
```

This seems to be their account for support and user feedback.
```suggestion
        "twitter": "MasterClassCare"
```